### DISCUSSION TITLE
Middleware for input fields?

### DISCUSSION THREAD

**Discussion by bartenra** [mirumee/ariadne#659] [Q&A] [answered]
I'm wondering if it's possible to add middleware / preprocessing for input fields?

My use case is that I want to add input field definition directives:

```graphql
input SomeInput {
  name: String!
  description: String! @html
}
```

And then run (in this case) `description` through a HTML sanitizer.

**Comment by rafalp** [comment, marked as answer]
Is there any reason you can't do it with custom scalar like `SanitizedString`?

**Reply by bartenra** [reply]
Excellent idea!
